ts/metrics/metric_type_enums.py

```diff
@@ -0,0 +1,10 @@
+"""
```

Comment:
nit: rename file `metric_type_enum.py` since there's just ever going to be one enum